scenarioo-client/app/components/detailarea/detail-accordion/detail-accordion.component.ts

```diff
@@ -22,50 +22,57 @@ import {SketchIdsResource} from '../../../shared/services/sketchIdsResource.serv
 import {RelatedIssueSummary} from '../../../shared/services/relatedIssueResource.service';
 import {LabelConfigurationMap} from '../../../shared/services/labelConfigurationsResource.service';
 import {ISketchIds} from '../../../generated-types/backend-types';
+import {LocalStorageService} from '../../../services/localStorage.service';
+
+const MAIN_METADATA_SECTION_EXPANDED = 'scenarioo-panelExpanded-';
```

Comment:
why "MAIN_" ?? - isnt it "LOCALSTORAGE_KEY_PREFIX_SECTION_EXPANDED" ?

Reply:
done